src/styles/utils/_fonts.scss

```diff
@@ -1,33 +1,17 @@
 @import 'mixins';
-//@import url('//fonts.googleapis.com/css?family=Roboto+Slab:400,300,100,700');
+@import '~system-font-css/system-font';
 
 $FONT_PATH: '//d2znry4lg8s0tq.cloudfront.net';
 
-$SECONDARY_FONT_KIEVIT: 'Helvetica';
-$THIRD_FONT_KIEVIT: 'arial';
-$FOURTH_FONT_KIEVIT: 'sans-serif';
+$SECONDARY_FONT_SYSTEM: 'Helvetica';
+$THIRD_FONT_SYSTEM: 'arial';
+$FOURTH_FONT_SYSTEM: 'sans-serif';
 
 $SECONDARY_FONT_MILO: 'serif';
 
 /* FONT DEFINITIONS
  * ---------------------------
 */
-/* KIEVIT INCLUDES */
-// @include font-face(Kievit-Regular, '#{$FONT_PATH}/ff_kievit_web/KievitWeb');
-/* Kievit Book */
-@include font-face(Kievit-Book, '#{$FONT_PATH}/ff_kievit_web/KievitWeb-Book');
-/* Kievit Book Italic */
-@include font-face(Kievit-Book-Italic, '#{$FONT_PATH}/ff_kievit_web/KievitWeb-BookIta');
-/* Kievit Meduim */
-@include font-face(Kievit-Medium, '#{$FONT_PATH}/ff_kievit_web/KievitWeb-Medi');
-/* Kievit Meduim Italic */
-@include font-face(Kievit-Medium-Italic, '#{$FONT_PATH}/ff_kievit_web/KievitWeb-MediIta');
-/* Kievit Bold */
-@include font-face(Kievit-Bold, '#{$FONT_PATH}/ff_kievit_web/KievitWeb-Bold');
-/* Kievit Light */
-// Used in the main Search Field Text only
-@include font-face(Kievit-Light, '#{$FONT_PATH}/ff_kievit_web/KievitWeb-Light');
-// @include font-face(Kievit-Thin, '#{$FONT_PATH}/ff_kievit_web/KievitWeb-Thin');
 
 /* KIEVIT Small Caps */
 //@include font-face(Kievit-SC, '#{$FONT_PATH}/ff_kievit_web/KievitScWeb');
```

Comment:
The comments used for example can also be updated (and lines 32-36)

Reply:
And lines 64-70.

Reply:
There's also one comment in the `_mixins` file that uses `Kievit-Book` as an example that can be removed.